.github/workflows/ci.yml

```diff
@@ -10,6 +10,7 @@ on:
       - "docs/**"
   repository_dispatch:
     types: check_connector_for_breaking_changes
+  workflow_dispatch:
```

Comment:
IINM we usually don't have manual triggers for the CI workflows in the other library repos.So any particular reason to have this?

Reply:
We have for the std-libs (in the master-build workflow). I need to include this to fix the `build` badge in the ASB repo.